src/components/TextInputWithPrefix/index.js

```diff
@@ -22,21 +23,18 @@ const propTypes = {
 
 const defaultProps = {
     errorText: '',
-    prefixCharacter: '',
     disabled: false,
     onChangeText: () => {},
 };
 
-const TextInputWithPrefix = props => (_.isEmpty(props.prefixCharacter)
-    // eslint-disable-next-line react/jsx-props-no-spreading
-    ? <TextInput {..._.omit(props, ['prefixCharacter', 'errorText'])} />
```

Comment:
We don't need this part anymore, it was here to make the TextInputWithLabel work without prefix characters anymore, but we don't need that anymore. This component will specifically only be needed when we have a `props.prefixCharacter`